### PR TITLE
Create ConfigurationInfo Page

### PR DESCRIPTION
**Pull request by ajaypratap003**
### Type of change

- Enhancement / new feature

### Description
1. Create ConfigurationInfo Page
2. Create StatusLabelWithIcon component
2. Create AdapterList componnent
3. Add story components for StatusLabelWithIcon, AdapterList, and ConfigurationInfo
4. Fix some warnings

image

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Resolve all warnings
- [x] Update license
- [ ] Write test cases for null checks in Unit Tests
- [ ] Reference relevant issue(s) and close them after merging

### CODE REVIEW

File: console/console-init/ui/src/components/AdapterList/AdapterList.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+};
+
+const Adapter: React.FC<{ adapter: IAdapter }> = ({ adapter }) => {
+  const { type, value } = adapter || {};
+  const { tlsEnabled, host, port, url } = value || {};
+
+  return (
+    <>
+      <Title size="xl" headingLevel="h2">
+        {type.toUpperCase() + " Adapter"}
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/components/AdapterList/AdapterList.tsx:52]: null | undefined check for type is missing

**Comment by anukritijha** [comment]
@ajaypratap003 As discussed, are all the warnings fixed?

**Comment by ajaypratap003** [comment]
> Looks good. A suggestion, the `StatusLabelWithIcon` component can be much more reusable if we can accommodate an `ExclamationCircleIcon` for disabled state somehow, as many of the pages need it. What do you think?
> 
> Screenshot from 2020-06-01 17-40-02

@rkpattnaik780 , `StatusLabelWithIcon` fully resuable. If you need different Icon and title you can pass as props. If you will not pass Icon and titiles as props it will take default Icons and titles.

File: console/console-init/ui/src/components/AdapterList/AdapterList.tsx (hunk 1 above)
Comment (ajaypratap003) [reply console/console-init/ui/src/components/AdapterList/AdapterList.tsx:52]: yes, I do. This change will come in next PR.

**Comment by rkpattnaik780** [comment]
> > Looks good. A suggestion, the `StatusLabelWithIcon` component can be much more reusable if we can accommodate an `ExclamationCircleIcon` for disabled state somehow, as many of the pages need it. What do you think?
> > Screenshot from 2020-06-01 17-40-02
> 
> @rkpattnaik780 , `StatusLabelWithIcon` fully resuable. If you need different Icon and title you can pass as props. If you will not pass Icon and titiles as props it will take default Icons and titles.

Okay, had missed that they are just default values.

**Comment by ajaypratap003** [comment]
> @ajaypratap003 As discussed, are all the warnings fixed?

@anukritijha , almost fixed, some are pending which will fix  after rebasing with master as you have already fixed loop related warings.

**Pull request merged**